src/Symfony/Component/Serializer/Normalizer/ObjectNormalizer.php

```diff
@@ -213,4 +213,21 @@ private function hasAttributeAccessorMethod(string $class, string $attribute): b
             && !$method->getAttributes(Ignore::class)
             && !$method->getNumberOfRequiredParameters();
     }
+
+    private function isReadableCache(object|string  $class, object|string $classOrObject, string $attribute): bool
```

Comment:
having `Cache` in the name does not make sense to me. This method should be name `isReadable` according to what it does. This method is not about caching at all.

Reply:
`isReadable is better`. I could not decide.

Reply:
I named the method `isReadableCache()` because the condition referred to `$isReadableCache`. But it is actually an `isReadable`.